### ISSUE TITLE
Error starting JSReport 2.0.0

### ISSUE THREAD

**Issue by brianmtully** [jsreport/jsreport-mongodb-store#6]
Starting jsreport 2.0.0 with jsreport-mongodb-store 1.0.1 it crashes with the following error:

```
2018-05-21T15:29:12.419Z - error: Error occured during reporter init TypeError: db.collection is not a function
    at Object.find (/project/node_modules/jsreport-mongodb-store/lib/provider.js:44:23
)
    at listenerPromise.then (/project/node_modules/jsreport-core/lib/store/collection.
js:65:66)
    at runCallback (timers.js:794:20)
    at tryOnImmediate (timers.js:752:5)
    at processImmediate [as _immediateCallback] (timers.js:729:5)

From previous event:
    at Object.then (/project/node_modules/jsreport-core/lib/store/collection.js:65:39)
(node:2730) UnhandledPromiseRejectionWarning: TypeError: db.collection is not a function
    at Object.find (/project/node_modules/jsreport-mongodb-store/lib/provider.js:44:23
)
    at listenerPromise.then (/project/node_modules/jsreport-core/lib/store/collection.
js:65:66)
    at runCallback (timers.js:794:20)
    at tryOnImmediate (timers.js:752:5)
    at processImmediate [as _immediateCallback] (timers.js:729:5)
From previous event:
    at Object.then (/project/jsreport-core/lib/store/collection.js:65:39)
```

**Comment by pofider** [comment]
Please share how do you configure this extension and what mongodb server version are you using.

**Comment by brianmtully** [comment]
Using mongodb V3.4.6 and the uri style configuration:

"store": {
  "provider": "mongodb"
},
"extensions": {
  "mongodb-store": {
    "uri": "mongodb://db1.example.net,db2.example.net:2500/?replicaSet=test"
  }
}

**Comment by pofider** [comment]
My apologies. There was error when using uri connection string.
This is fixed in jsreport-mongodb-store@1.0.2
